google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -59,7 +59,9 @@ void CreateInstance(google::cloud::bigtable::InstanceAdmin instance_admin,
   google::cloud::bigtable::DisplayName display_name("Put description here");
   std::string cluster_id = instance_id + "-c1";
   auto cluster_config = google::cloud::bigtable::ClusterConfig(
-      zone, 3, google::cloud::bigtable::ClusterConfig::HDD);
+      google::cloud::bigtable::ProjectId(instance_admin.project_id()),
```

Comment:
Can we avoid passing the `project_id` as an explicit argument?  If the user can *only* provide the same project id that is embedded in the `instance_admin` object, why force them to provide that value? It seems obvious to me that we should just reuse it in the member functions that consume `ClusterConfig` objects, but maybe I missed something?

Reply:
ACK. 
I will revert the changes and modify location in `InstanceAdmin::CreateClusterImpl` where `ClusterConfig` is consumed.